### ISSUE TITLE
can it run XXNoFrameskip-v4?

### ISSUE THREAD

**Issue by THSWind** [grantsrb/PyTorch-A2C#2]

**Comment by grantsrb** [comment]
Not sure, but probably. You would need to change the `hyperparams.json` value for `env_name` to whatever env you want to use, and then change the value for `prep_fxn` to either "normalize_prep" or whatever you define in `preprocessing.py`.

**Comment by grantsrb** [comment]
I just added an example `hyperparams.json` file. Hopefully that helps.

**Comment by THSWind** [comment]
okay, thanks for your reply~

**Comment by THSWind** [comment]
> I just added an example `hyperparams.json` file. Hopefully that helps.

Hi grantsrb,
     I'm puzzled by your hyperparams.json file,  I don't know what's the  meanning of it's elements, and how to set it, just like 'prep_fxn','normalize_prep'. would you mind to adding a discord friend? if you use it and convenience, thanks for your kindness~

**Comment by THSWind** [comment]
> I just added an example `hyperparams.json` file. Hopefully that helps.

https://discord.gg/txj5VaFN

**Comment by THSWind** [comment]
Hello, grantsrb,
I want to reproduce your repo,  while I found your's usage in readme is doesn't work. I'm failed. is there any problem?

**Comment by THSWind** [comment]
![image](https://user-images.githubusercontent.com/42888230/104469183-3d5f4600-55f3-11eb-9184-8a1885525e85.png)

**Comment by THSWind** [comment]
I have tried many times in many ways. could you please  help me to reproduce it?

**Comment by grantsrb** [comment]
Yeah, that was due to the dependencies listed in a custom repo. Thanks for pointing it out.

Try running the following from the top directory of this cloned repo:
`$ python3.6 -m pip install --upgrade pip`
`$ python3.6 -m pip install -r requirements.txt`

**Comment by THSWind** [comment]
It doesn't works~
![image](https://user-images.githubusercontent.com/42888230/104560264-99bc7700-5680-11eb-8aab-a347bcd25b9a.png)

**Comment by THSWind** [comment]
it works, but strangely is: no matter how I edit(online or in vscode), this error occurs still~
![image](https://user-images.githubusercontent.com/42888230/104565181-e014d480-5686-11eb-923f-2ae19e8fa499.png)

**Comment by THSWind** [comment]
> it works, but strangely is: no matter how I edit(online or in vscode), this error occurs still~
> ![image](https://user-images.githubusercontent.com/42888230/104565181-e014d480-5686-11eb-923f-2ae19e8fa499.png)

I have dealt with it, maybe because you have use illegal character("blank" character)

**Comment by THSWind** [comment]
It seems can't run "BreakoutNoFrameskip-v4", what a pity, thanks for your help~
